src/frontend/metrics/getPercentageOfNetworkAvailability.ts

```diff
@@ -0,0 +1,10 @@
+import type {Observation} from '@mapeo/schema';
+
+export default function (observations: ReadonlyArray<Observation>) {
+  const networkAvailability = observations
+    .map(obs => obs.metadata?.positionProvider?.networkAvailable)
+    .filter(networkAvailable => networkAvailable !== undefined);
+
+  const trues = networkAvailability.filter(Boolean);
+  return (trues.length / networkAvailability.length) * 100;
```

Comment:
This could result in a division by 0. Could we check for this case and return 0 or maybe `null`?